testgrid/config/config.proto

```diff
@@ -137,6 +137,10 @@ message DashboardTab {
   // Default code search path for changelist search links
   string code_search_path = 4;
 
+  // Base options to always include, for example:
+  // width=20&include-filter-by-regex=level_tests
```

Comment:
maybe include what are valid options that can be configured, like what does `include-filter-by-regex=level_tests` mean.